ansible/roles/rke/tasks/ssl-certificates.yml

```diff
@@ -1,54 +1,25 @@
 ---
```

Comment:
@steved @Secretions I believe this commit addresses the concerns from today's discussion. The certificates are generated locally and avoids any remote package installations. 

Note, it still uses OpenSSL mainly because that's the path of least resistance.